Updated sample after fixing BUG.

jparrent opened the pull request.

guillochon commented:
What was the reason for updating the HTML files, out of curiosity? Was it because the new spectra were only recently uploaded to WISeREP?

jparrent commented:
Yeah those were for the recent additions. It's primarily because of how I initially birthed the sub-directories, by querying WISEREP for observations taken between YYYY-01-01 and YYYY-12-31 (instead of the much smarter way of when the observations were uploaded). At the time I was interested in yearly statistics. And because the data, e.g., for 96al was obtained in 96, 97, 98, 02, 04, 07, and 11, those pages were updated to suck up the 96al data. And the same for the others. At some point this summer I think it would be reasonable for me to restructure the yearly chunks as they were uploaded. Would be much easier to update.

guillochon commented:
OK, attempting an import with this PR, will let you know if successful. For reference we have 14,026 spectra in total before the PR, so hopefully that number goes up after the import. Should know by tomorrow morning.

guillochon commented:
So one issue here: We have to avoid git LFS, they impose a bandwidth limit per month of only 1 GB (something I was not aware of originally). There's one file in this PR that exceeds 100 MB (20010101_20101231/WISEREP_20010101_20101231.html), any way that we can split this file into multiple so its size doesn't exceed 100 MB?

jparrent commented:
Ah that's right, I forgot to mention and also address this. Yeah I can split that no problem. Do you prefer canceling the PR while I get to that, or continuing and having me update after the merge? Either works for me.

guillochon commented:
Go ahead and update the PR, I will pull and update again later.

jparrent commented:
Updated my sne-external-WISEREP with the split pages. 
WISEREP_20010101_20051231 (55.7 Mb)
WISEREP_20060101_20101231 (55.4 Mb)

guillochon commented:
So before the file split we dropped by ~2000 spectra, so something is wrong with the updated html (only 11,914 in the catalog with this PR). Resetting for now, but I'm guessing file split version may have same problem.

jparrent commented:
That's interesting. The 2001-2010 directory alone should have 1918 spectra. Let me know what happens after using the split pages and I can investigate further.

guillochon commented:
My guess would be a change in the HTML file format from the previous way we saved the HTML file. I'll try with the split files but my guess will be the outcome will be the same. Ideally we should try and do this in a more stable way in the future.

jparrent commented:
Agreed. I will be in the office Thursday, and can be in on Friday too, to take a closer look at the html issue. "Nothing" changed in how I grabbed the html files between then and now, so I'm not sure what's to blame.

guillochon commented:
OK great, we'll try to figure it out then. I think we should try to keep things as stable as possible over the next week or so.

jparrent commented:
Yeah this was intended to be the last batch until the dust settles. Feel free to delay the PR if it's only going to reduce the number of spectra until we figure it out.

jparrent commented:
@guillochon Hold please. Found the wrong html page for Year:2000.

jparrent commented:
Updated.

guillochon commented:
Tried merging this PR again, this time only 11,892 after running the import script. So definitely something up with the new HTML files.

jparrent commented:
I'm looking at the old and new pages for Year 2000 and I believe I see what the issue is. I will work on this in the morning and report back.

jparrent commented:
Alright, back again. I have updated the pages using Safari since this seems to give the source in the proper format. I have prefixed the .html files with "safari" to keep track of what's up, but have removed the other html files. I have also added the LSQ12dlf Sept 12 spectrum and the SN2015bn (PS15ae) spectra to their appropriate folders. 

If this does not work, I can try one more option. I hope the route of Sarafi works. Chrome is junk (the previous round), and Firefox is a pain for this pursuit, but can be done. Otherwise back to the drawing board.

guillochon commented:
OK testing this one. Will not make it live on the web though this time, will create script to count the spectra before making it live. Should do this anyway to avoid borking everything on the live website. This will also test if the revised SNDB import is working.

guillochon commented:
So down to 13,039 now, so some of the spectra came back with this latest PR.

Let's chat on Friday to try and resolve.

jparrent commented:
Good, better news. I will look at one query page from all browsers in the meantime to see what gives.

Commit:
Updated sample after fixing BUG.
